src/main/java/ladder/view/OutputView.java

```diff
@@ -0,0 +1,29 @@
+package ladder.view;
+
+import java.util.List;
+
+public class OutputView {
+    public void printNames(List<String> names) {
+        System.out.println("실행결과");
+        System.out.println();
+
+        for(String name : names) {
+            System.out.print(String.format("%5s ", name));
+        }
+        System.out.println();
+    }
+
+    public void printLadders(List<List<Boolean>> ladders) {
+        for(List<Boolean> ladder : ladders) {
+            System.out.print("    ");
+            for(Boolean connected : ladder) {
+                if(connected) {
+                    System.out.print("|-----");
+                } else {
+                    System.out.print("|     ");
+                }
+            }
+            System.out.println("|");
+        }
+    }
```

Comment:
depth가 깊어보이는데 스트림과 메소드 추출을 활용하면 좋겠습니다!

```suggestion
    public void printLadders(List<List<Boolean>> ladders) {
        for (List<Boolean> ladder : ladders) {
            var lines = ladder.stream()
                    .map(this::make)
                    .collect(Collectors.joining("", "    ", "|"));
            System.out.println(lines);
        }
    }

    private String make(boolean connected) {
        if (connected) {
            return "|-----";
        }
        return "|     ";
    }
```

메소드 이름은 예시이니 참고만 해주세요!